core/commands/publish.go

```diff
@@ -102,7 +103,7 @@ Publish an <ipfs-path> to another public key (not implemented):
 	Marshalers: cmds.MarshalerMap{
 		cmds.Text: func(res cmds.Response) (io.Reader, error) {
 			v := res.Output().(*IpnsEntry)
-			s := fmt.Sprintf("Published name %s to %s\n", v.Name, v.Value)
+			s := fmt.Sprintf("Published %s to %s\n", v.Value, v.Name)
```

Comment:
On Thu, May 14, 2015 at 09:24:16PM -0700, Juan Batiz-Benet wrote:
> > -			s := fmt.Sprintf("Published name %s to %s\n", v.Name, v.Value)
> > +			s := fmt.Sprintf("Published %s to %s\n", v.Value, v.Name)
> 
> this order is flipped from the input…

We're currently not doc'ing this consistently (see the commit message
for 7e9db663, core/commands/publish: Fix published message,
2015-05-07).  To preserve the argv order, how about:

  $ ipfs name publish <name> <value>
  Published to <name>: <value>

?